examples/r-job/README.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Create a Job Using R"
+weight: 2
+description: "Setting and executing a job on Saturn Cloud using R"
+---
+
+
+A Saturn Cloud job is a computing environment set up to run recurring tasks. You can start a job in 3 ways:
+
+1. on a schedule 
+2. manually by pressing start button 
+3. via an API 
+
+## Setting up the Saturn Cloud Job Resource
```

Comment:
I don't think you need a section title for this, it can just be part of the overview

```suggestion
```